docs/Android/privacy-config.md

```diff
@@ -57,36 +60,39 @@ public static void setPrivacyConfiguration(Map<String, Object> privacyConfig, @N
-|                     |                                                                                     |      AVAILABLE AND EMPTY     | All attributes are computed and reported for the corresponding entity.                                |                                                                                                                                                        |
-|                     |                                                                                     |    AVAILABLE AND NOT EMPTY   | Attributes in the array are disabled from being computed or transmitted for the corresponding entity. |                                                                                                                                                        |
-|   EventsToDisable   |                 Used for disabling specific events within an entity                 |                              |                                                                                                       |                                                                                                                                                        |
-|                     |                                                                                     |            MISSING           | All events are computed and reported for the corresponding entity.                                    |                                                                                                                                                        |
-|                     |                                                                                     |      AVAILABLE AND EMPTY     | All events are computed and reported for the corresponding entity.                                    |                                                                                                                                                        |
-|                     |                                                                                     |    AVAILABLE AND NOT EMPTY   | Events in the array are disabled from being computed or transmitted for the corresponding entity.     |                                                                                                                                                        |
+|      **Flag**       | **Description**                                                                                           |          **Value**           |                                              **Outcome**                                              |                                                                       **Notes**                                                                        |
```

Comment:
Please add the following within the table - 
* In the `outcome` column for `AppUsageData` , when the value for the key is `available`, let's update the content to say - `app_usage_metrics attributes are all computed / disabled as per the configured flag ` 
* In the row for `AttributesToDisable `, let's add another note in the `notes` column that attributes from `app_usage_metrics` entity cannot be individually enabled/disabled. It is all or none (Please re-phrase as required)